### PR TITLE
Kubetest2 fix periodic end to end tests

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: tests/e2e/kubetest2-kops/aws/zones.go [hunk 1]
```diff
@@ -75,7 +75,6 @@ var allZones = []string{
 	"us-east-2a",
 	"us-east-2b",
 	"us-east-2c",
-	"us-west-1a",
 	"us-west-1b",
```
Comment (hakman) [inline tests/e2e/kubetest2-kops/aws/zones.go:78]: Please just comment these same as the one below. We can live without any `us-west-1`.

**Review by hakman — CHANGES_REQUESTED** [review]

File: tests/e2e/kubetest2-kops/aws/zones.go (hunk 1 above)
Comment (olemarkus) [reply tests/e2e/kubetest2-kops/aws/zones.go:78]: Is there a benefit to comment them out?

Comment (hakman) [reply tests/e2e/kubetest2-kops/aws/zones.go:78]: Yes. At some point someone will wonder why there's no `us-west-1` and he will add it, then remove it again.
This way maybe he reads the comment first :D.

Comment (olemarkus) [reply tests/e2e/kubetest2-kops/aws/zones.go]: For me 1c exists. It probably depends on the account which one gets cut.
I'll leave this change out for now.

**Review by hakman — APPROVED** [review]

**Comment by olemarkus** [comment]
Shouldn't that test have been skipped?

**Comment by rifelpet** [comment]
In the ginkgo output i see:

```
volume snapshot controller  [It] should check snapshot fields
```

Which doesn't exactly match the skip regex. Maybe drop the "volume.snapshot.controller" portion.

**Comment by olemarkus** [comment]
I thought some of those bits were only used when pretty printing. not actually used for the regex. But apparently I was wrong there.

**Pull request merged**